FraudDetectionUsingGraphAnalytics.ipynb

```diff
@@ -269,10 +269,41 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Connect and Create a Graph"
+    "## Connect and Create a Graph\n",
+    "\n",
+    "In this step, we will connect to the ADB using the Graph Client Python libraries `opg4py` and `pypgx`, so you can create graphs on memory and execute PGQL queries against them.\n",
+    "\n",
+    "To connect to the ADB, please prepare the information below:\n",
```

Comment:
@ryotayamanaka In the OCI marketplace installation, we actually populate these environment variables automatically for the user so they don't have to prepare this information. Could you remove these lines? Otherwise this looks good to me. Thanks!

Reply:
Thank you for your review. I edited line 280 as suggested. Thanks!